.github/workflows/gradle.yml

```diff
@@ -0,0 +1,20 @@
+name: Gradle Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Gradle
+      run: ./gradlew -Pversion=$GITHUB_REF release --scan --no-daemon -i
```

Comment:
I am a bit worried that we are not running the tests on release (we did on Travis)

Reply:
Fixing.